dev/docker/ocis.web.config.json

```diff
@@ -27,7 +27,6 @@
   "apps": [
     "files",
     "text-editor",
-    "preview",
```

Comment:
for more context on why this is removed https://github.com/owncloud/ocis/issues/4771#issuecomment-1273222856